### PR TITLE
[EMB-408] Institutions landing page and quickfiles tests

### PR DESCRIPTION
**Pull request by brianjgeiger**
## Purpose

Test the institutions landing page and quickfiles

## Summary of Changes

1. Moved some static images into our repo
2. Tests!
3. Removed a redundant button on the rename conflict dialog and desaturated the buttons
4. Upgrade ember-css-modules-sass from 1.0.0 to 1.0.1 to eliminate deprecation warning

## Ticket

https://openscience.atlassian.net/browse/EMB-408

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] testable and includes test(s)
- [ ] ~~changes described in `CHANGELOG.md`~~

### CODE REVIEW

File: tests/acceptance/guid-user/quickfiles-test.ts [hunk 1]
```diff
@@ -37,227 +48,600 @@ module('Acceptance | Guid User Quickfiles', hooks => {
...
+            const currentUser = server.create('user', 'loggedIn', 'withFiles');
+
+            await visit(`--user/${currentUser.id}/quickfiles`);
+            assert.dom('[data-test-file-item-link]').exists({ count: 5 }, 'initial state');
+
+            await untrackedClick('[data-test-download-count]');
+            assert.dom('[data-test-delete-file-button]').exists('after select file');
+
+            await click('[data-test-delete-file-button]');
+            assert.dom('[data-test-delete-modal]').exists('after click delete');
```
Comment (jamescdavis) [inline tests/acceptance/guid-user/quickfiles-test.ts]: percySnapshot()

File: tests/acceptance/guid-user/quickfiles-test.ts [hunk 2]
```diff
@@ -37,227 +48,600 @@ module('Acceptance | Guid User Quickfiles', hooks => {
...
+            );
+
+            await visit(`--user/${currentUser.id}/quickfiles`);
+            assert.dom('[data-test-file-item-link]')
+                .exists({ count: 2 }, 'initial state');
+            assert.dom('[data-test-file-item-link]')
+                .containsText(fileTwo.name, 'initial state');
+            assert.dom('[data-test-conflict-keep-both-button]')
+                .doesNotExist('initial state');
+            await untrackedClick('[data-test-download-count]');
```
Comment (jamescdavis) [inline tests/acceptance/guid-user/quickfiles-test.ts]: Let's be more definitive on which file we're selecting.

File: tests/acceptance/guid-user/quickfiles-test.ts [hunk 3]
```diff
@@ -37,227 +48,600 @@ module('Acceptance | Guid User Quickfiles', hooks => {
...
+                {
+                    guid: 'xyzzy',
+                    name: 'Xyzzyplugh.gif',
+                    user: currentUser,
+                },
+            );
+            await visit(`--user/${currentUser.id}/quickfiles`);
+            assert.dom('[data-test-file-item-link]').exists({ count: 1 });
+
+            await untrackedClick('[data-test-download-count]');
```
Comment (jamescdavis) [inline tests/acceptance/guid-user/quickfiles-test.ts]: Add comment about what this does.

File: tests/acceptance/institutions-test.ts [hunk 4]
```diff
@@ -1,19 +1,78 @@
...
+        assert.dom('[data-test-visit-institution]').exists({ count: 1 });
+        await fillIn('[data-test-institutions-filter]', 'aa');
+        assert.dom('[data-test-visit-institution]').exists({ count: 1 });
+        await fillIn('[data-test-institutions-filter]', 'zz');
+        assert.dom('[data-test-visit-institution]').exists({ count: 1 });
+        await fillIn('[data-test-institutions-filter]', 'z');
+        assert.dom('[data-test-visit-institution]').exists({ count: 3 });
+    });
+
+    test('can filter by name', async assert => {
```
Comment (jamescdavis) [inline tests/acceptance/institutions-test.ts]:
```suggestion
    test('can sort by name', async assert => {
```

**Comment by brianjgeiger** [comment]
@jamescdavis Okay, this is all responded to.

**Pull request merged**